src/structures/interfaces/InteractionResponses.js

```diff
@@ -11,14 +11,14 @@ const MessagePayload = require('../MessagePayload');
  */
 class InteractionResponses {
   /**
-   * Options for deferring the reply to an {@link Interaction}.
+   * Options for deferring the reply to a {@link MessageComponentInteraction}.
```

Comment:
Why

Reply:
Interaction can be both select menu interaction and button interaction but here only button interaction was mentioned

Reply:
No, this mentioned Interaction and is still correct, please revert this

Reply:
Oh my bad I'll close the pr

Reply:
No, just revert this line alone

Reply:
yeah sorry i misspelled